app/split_integer.py

```diff
@@ -5,3 +5,6 @@ def split_integer(value: int, number_of_parts: int) -> list:
         parts.append(value // parts_left)
         value -= next_number
     return parts
+
+
+print(split_integer(4,5))
```

Comment:
The function call `split_integer(4,5)` will result in a list with zero or negative values because the number of parts (5) is greater than the integer value (4). Consider adding a check to ensure `number_of_parts` does not exceed `value` or handle the case where it does.